Source/Core/CommonStats.cpp

```diff
@@ -46,9 +46,6 @@ CommonStats::CommonStats (const struct per_item* PerItem_, int Type_, size_t Cou
     additionalIntStats(nullptr),
     additionalStringStats(nullptr)
 {
-    // Adaptation for having a graph even with 1 frame
-    if (FrameCount<2)
-        FrameCount=2;
```

Comment:
Does a single frame video (such as a static image file), produce any graph after this change?

Reply:
So you say that was required purely to produce chart? To make two dots from one?
if that was the intention, let me think how to return that functionality without breaking filters

Reply:
On this branch, QCTools gui crashes when opening a jpg.

Reply:
Jpeg issue should be resolved